### DISCUSSION TITLE
Add option to block creating maps inside terrains

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [GriefPrevention/GriefPrevention#1451] [Ideas]
It would be interesting to have this option in the plugin, sometimes they want to make map-art and there is no way to protect

**Comment by Jikoo** [comment]
Teach your players about the vanilla feature for locking maps using a cartography table.

**Reply by thiagogebrimm** [reply]
I want to block that I can make a map inside other terrains

**Reply by Jikoo** [reply]
Can you present an actual use case for blocking map creation that is not solved by unmodifiable maps?
